rust/prql/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "_prql_library"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate_type = ["staticlib"]
+doctest = false
+test = false
+
+[dependencies]
+prql-compiler = "0.4.0"
```

Comment:
Just a question: is there any reason to use this version? I believe crates.io currently has 0.8.1.